distributions/distribution-resources/src/main/resources/conf/services/readme.txt

```diff
@@ -0,0 +1,8 @@
+
+openHAB2 conf/services folder
+
+Your service configurations will reside here.
+All configuration files have to have the ".cfg" file extension.
+Service configuration files will be automatically created as soon as you install a binding/ui/persistence/... that needs further configuration.
```

Comment:
"... are automatically created ..."
"... install an add-on that can be configured."